packages/flutter/lib/src/widgets/scroll_controller.dart

```diff
@@ -88,7 +88,6 @@ class ScrollController extends ChangeNotifier {
   ///
   /// This should not be mutated directly. [ScrollPosition] objects can be added
   /// and removed using [attach] and [detach].
-  @protected
```

Comment:
Ooo this is a very exciting change. There are so many work arounds for this that we could clean up. :)